startupscript/butane/aws/metadata-utils.sh

```diff
@@ -49,17 +49,18 @@ readonly -f get_guest_attribute
 function set_metadata() {
   local key="${1}"
   local value="${2}"
-  
+  local escaped="${value//[\"\\]/\\&}"
```

Comment:
Please add a comment indicating the substitution that this command is doing and why (point to AWS documentation if it exists).

Reply:
done